src/test/java/com/conveyal/r5/analyst/network/SimpsonDesertTests.java

```diff
@@ -146,6 +146,13 @@ public void testGridFrequencyAlternatives () throws Exception {
         DistributionTester.assertExpectedDistribution(twoAlternatives, travelTimePercentiles);
     }
 
+    /**
+     * For evaluating results from the tests below.
+     */
+    private int[] roundPathTimesToMinutes (PathResult.PathIterations paths) {
+        return paths.iterations.stream().mapToInt(i -> (int) (Math.round(i.totalTime / 60f * 10) / 10.0)).toArray();
```

Comment:
It makes sense to factor out that repeated stream operation, and round rather than truncating when comparing against a list of estimates in minutes. But what's the thinking behind the multiplication and division by 10? If I'm not mistaken this is rounding to the nearest 1/10 minute, then shifting the decimal point back before truncating, giving a similar result to just truncating the original value except for input values within 0.05 below an integer. If this is the intended behavior the purpose should probably be explained in the javadoc. Also, minor detail: floating point literals in Java are double precision by default, so there are extra type conversions between float and double in this series of multiplications and divisions. In places where we don't need to minimize storage space it's typical to just use doubles (d suffix or no suffix) instead of single-precision floats (f suffix).

Reply:
I do not know the original thinking behind how or why it was written this way. This code was copied from the original implementation here: 

https://github.com/conveyal/r5/blob/a528745a51c5b5e9a8d975ffb6f94cafe601bfe4/src/main/java/com/conveyal/r5/analyst/cluster/PathResult.java#L230-L232

The `SimpsonDesertTests` were written to compare against those results. But after this change, the API no longer uses the `HumanReadableIteration` class or its results. So instead of keeping that class I just moved the transform to this new method within the `SimpsonDesertTests`.

Reply:
That makes sense. I think you mentioned this change the other day and I now see what you mean, this is just composing the old (now unused) function with the stream operation that was applied to it in the test. The old function was just rounding to the nearest tenth of a minute for display purposes (hence "human readable"). These happened to be the values readily available for comparison in the test. Once the two steps are combined though with no use for human-friendly display rounding, it creates an expression that can be confusing to read later - it had me thinking there must be some subtle trick or nuance to this combination of conversions, multiplication, and division.

Now that we don't need rounded human readable values, what we're really after here is an equality check with tolerance ("epsilon"). The testing framework should supply this, I'll see if I can track it down and replace it.

Reply:
Patch at https://github.com/conveyal/r5/pull/830